generators/client/templates/react/src/main/webapp/app/entities/index.tsx.ejs

```diff
@@ -18,7 +18,9 @@
 -%>
 import * as React from 'react';
 // tslint:disable-next-line:no-unused-variable
```

Comment:
you need to move the `tslint:disable-next-line:no-unused-variable` below so that it is applied to `ErrorBoundaryRoute` so that linting doesnt fail when there are no routes here on a fresh app without entities